src/components/Button/Button.css

```diff
@@ -1,3 +1,30 @@
 .Button {
 	padding: 1em;
+	padding-left: 2em;
+	padding-right: 2em;
+	border: none;
+	border-radius: 4px;
+	font-size: 15px;
+	font-weight: bold;
+
+}
```

Comment:
Few things
- ``padding: 1em; `` is now half-redundant because of ``padding-left`` and ``padding-right``, you should be able to squish them all into short hand by doing ``padding: 1em 2em;``
- You use ``em`` for the dimensions but then ``px`` for the ``font-size``, it's probably safer to set these all as ``px`` for the time being (for reference ``em`` is equal to the font size)